contracts/ZkSync_SpokePool.sol

```diff
@@ -33,18 +34,39 @@ contract ZkSync_SpokePool is SpokePool {
 
     // Bridge used to withdraw ERC20's to L1
     ZkBridgeLike public zkErc20Bridge;
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    ZkBridgeLike public immutable zkUSDCBridge;
```

Comment:
Why make this immutable and not the same as `zkErc20Bridge`? Also I don't see this set in the constructor or the initializer